apps/playground-web/src/app/connect/auth/actions/auth.ts

```diff
@@ -0,0 +1,49 @@
+"use server";
+import { THIRDWEB_CLIENT } from "@/lib/client";
+import { cookies } from "next/headers";
+import { type VerifyLoginPayloadParams, createAuth } from "thirdweb/auth";
+import { privateKeyToAccount } from "thirdweb/wallets";
+
+// FIXME TODO Somehow Vercel is not able to get the `THIRDWEB_ADMIN_PRIVATE_KEY` during build
+// should never have to do this (im sorry)
+const privateKey =
+  process.env.THIRDWEB_ADMIN_PRIVATE_KEY ||
+  process.env.NEXT_PUBLIC_THIRDWEB_ADMIN_PRIVATE_KEY;
```

Comment:
How about we just generate an account in the fallback case

Reply:
yoooo that is a great idea